package.json

```diff
@@ -20,9 +20,19 @@
   },
   "homepage": "https://github.com/cs2340-group3/moviebuzz",
   "dependencies": {
+    "bcrypt-nodejs": "0.0.3",
+    "body-parser": "^1.12.4",
+    "cookie-parser": "^1.3.5",
+    "csurf": "^1.8.2",
     "express": "4.12.4",
-    "passport": "0.2.2",
-    "passport-local": "1.0.0",
+    "express-handlebars": "^2.0.1",
+    "express-session": "^1.11.2",
+    "handlebars": "^3.0.3",
+    "mongoose": "^4.0.4",
+    "morgan": "^1.5.3",
+    "passport": "^0.2.2",
+    "passport-local": "^1.0.0",
+    "passport-local-mongoose": "^1.0.0",
```

Comment:
I'd favor using exact versions (`1.0.0`) rather than semver ranges (`^1.0.0`). Using the latter can lead to unreproducibility later if package authors make mistakes in their versioning (i.e. if they make a breaking change but don't bump the major version number).